docs/pages/server-access/getting-started.mdx

```diff
@@ -55,11 +56,23 @@ This guide introduces some of these common scenarios and how to interact with Te
 
 Next, create a join token so you can add the Node to your Teleport cluster.
 
+<ScopedBlock scope={["oss", "enterprise"]}>
+
 ```code
 # Let's save the token to a file
-$ sudo tctl tokens add --type=node | grep -oP '(?<=token:\s).*' > token.file
+$ sudo tctl tokens add --type=node --format json | jq -r '.token'  > token.file
```

Comment:
Is the only difference between these `ScopedBlock`s the presence of `sudo` for self-hosted clusters? I'm not sure we want to assume that users who self-host will want to execute `tctl` commands on the Auth Service host. For those who do, I think the `permission denied` error message would add an acceptable amount of friction—what do you think?

Reply:
consolidated